external/model-preparation-algorithm/mpa_tasks/apis/config.py

```diff
@@ -117,21 +118,66 @@ class BaseLearningParameters(ParameterGroup):
             max_value=1000,
             header="Iteration patience for early stopping",
             description="Training will stop if the model does not improve within the number of iterations of patience. \
-                        This ensures the model is trained enough with the number of iterations of patience before early stopping.",
+                        the model is trained enough with the number of iterations of patience before early stopping.",
```

Comment:
Early stopping instances in config.py is duplicated. Please remove the duplicated code or checkout config.py file and resolve pre-commit again.

Reply:
I fixed it.